betty/plugin/config.py

```diff
@@ -131,3 +138,85 @@ def load_item(self, dump: Dump) -> PluginConfiguration:
     @classmethod
     def _create_default_item(cls, configuration_key: str) -> PluginConfiguration:
         return PluginConfiguration(configuration_key, {})
+
+
+async def ensure_plugin_configuration(
```

Comment:
Raise a user facing error if the extension is configurable, cannot produce its own default config, and no config was given.

Reply:
Make are this is tied into config validation/loading